paper-fab.html

```diff
@@ -147,7 +148,24 @@
           type: Boolean,
           value: false,
           reflectToAttribute: true
+        },
+        
+        /**
+         * The label displayed in the badge. The label is centered, and ideally
+         * should have very few characters.
+         */
+        label: {
+          type: String,
+          observer: '_labelChanged'
         }
+      },
+      
+      _labelChanged: function() {
+        this.setAttribute('aria-label', this.label);
+      },
+      
+      _computeIsIconFab: function(icon) {
```

Comment:
Could you please add a test to see that this works? I.e. a test that if the label is set, it's being used, and if a label AND an icon are set, the icon is displayed?